Cargo.toml

```diff
@@ -60,7 +60,7 @@ rev = "70b0eb5"
 features = ["stm32h743v", "rt", "unproven", "ethernet", "quadspi"]
 # version = "0.9.0"
 git = "https://github.com/quartiq/stm32h7xx-hal.git"
-rev = "acd47be"
+rev = "33aa67d"
```

Comment:
This is actually the commit that utilizes `master` smoltcp. I erroneously added that comment in the last PR.